app/controllers/api/console.php

```diff
@@ -38,3 +39,57 @@
 
         $response->dynamic($variables, Response::MODEL_CONSOLE_VARIABLES);
     });
+
+
+App::post('/v1/console/assistant')
+    ->desc('Ask Query')
+    ->groups(['api', 'assistant'])
+    ->label('scope', 'public')
+    ->label('sdk.namespace', 'assistant')
+    ->label('sdk.method', 'chat')
+    ->label('sdk.description', '/docs/references/assistant/chat.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_TEXT)
+    ->param('query', '', new Text(2000), 'Query')
+    ->inject('response')
+    ->action(function (string $query, Response $response) {
+        $ch = curl_init('http://appwrite-assistant:3003/');
+        $responseHeaders = [];
+        $query = json_encode(['prompt' => $query]);
+
+        $headers = ['accept: text/event-stream', 'authorization: ' . APP::getEnv('ASSISTANT_SECRET')];
```

Comment:
Why do we need this secret? This container is going to be accessible only from private network

Reply:
I think we can skip it, for the executor it was to protect different runtimes from reaching each other.